features/visitor_can_only_view_part_of_premium_article.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Visitor can only view part of premium article
```

Comment:
Add a sad path for a non subscriber trying to access the premium article.

Reply:
That's the whole scenario? 
We have a visitor that tries to access a premium article. He get's a notice that he needs to subscribe to access this. 

Next feature (with stripe) is to become a subscriber and get access to the content.

Reply:
Yes, sry did not read it properly the first time, but going over it again this story starts to feel off, for instance `"To read this premium article, you need to become a subscriber."` is more of a sad path for the subscribe feature.

Reply:
I feel the user story it self it's not really "user friendly" since it's purely a locking of content. 
Don't know how we could do this as a Happy Path, maybe this is not a feature in that sense. 
Maybe this one should have been a chore?

Reply:
No, the story could have been defined better, when reading it I just get the impression of only showing the premium article title and lead not about restricting the premium article show page.